utils/src/main/scala/cromwell/pipeline/utils/json/DummyObject.scala

```diff
@@ -0,0 +1,9 @@
+package cromwell.pipeline.utils.json
```

Comment:
I think we can place this class in `utils` package instead of `utils.json`. There is nothing specific for `json`.

Reply:
done, moved